modules/ztp-image-based-upgrade-procedure-rollback.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-procedure-rollback_{context}"]
+= Rolling back an image-based upgrade on managed clusters at scale
+
+Roll back the changes on a set of managed clusters if you encounter unresolvable issues after a successful upgrade.
+You need to create a separate `ImageBasedGroupUpgrade` CR and define the set of managed clusters that you want to roll back.
```

Comment:
```suggestion
You need to create a separate `ImageBasedGroupUpgrade` CR and define the set of managed clusters that you want to roll back.
```

Separate from? Add a line before the paragraph or just join it to the previous paragraph.